app.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'sinatra/reloader'
+require 'json'
+
+helpers do
+  def h(text)
+    Rack::Utils.escape_html(text)
+  end
+end
+
+def create_memo(memo)
+  File.open("./memos/#{memo[:id]}.json", 'w') do |file|
+    file.puts(memo.to_json)
+  end
+end
+
+def read_memo
```

Comment:
`params`もどこからでもアクセスできるという点で一種のインスタンス変数のようなものなので、このメソッドは引数を与えた方が再利用性の高いメソッドになると思います。

```suggestion
def read_memo(id)
```

Reply:
>paramsもどこからでもアクセスできるという点で一種のインスタンス変数のようなものなので

おぉ、その考え方はありませんでした😮
修正しました✨